src/modules/UI/components/Buttons/PrimaryButtonScaled.ui.js

```diff
@@ -0,0 +1,39 @@
+// @flow
```

Comment:
Put scaling into PrimaryButton and remove use of Scaled versions of components. We're looking to get the full app scaled now